supervisord.conf

```diff
@@ -1,6 +1,6 @@
 [program:smarts]
 environment=PYTHONUNBUFFERED=1
-command=python examples/single_agent.py scenarios/intersections/6lane
+command=python examples/single_agent.py scenarios/loop
```

Comment:
Is it intentional that we want to set the default scenario to loop?

Reply:
I reverted.